README.md

```diff
@@ -1,10 +1,6 @@
-<a href="https://github.com/unfoldingWord/findr/" >
-  <img alt="Findr hero image" src="images/findr-cover.png">
-</a>
-
 # findr
 
-A set of **Open Component Packages** for creating fast and highly configurable **find-and-replace** user interfaces with javascript.
+findr is a set of utility libraries for building find and replace UIs in React.
```

Comment:
Ah, things make more sense now. I'll re-write this and the `## Support` section with this in mind. What I could use from you is: a statement about each existing package. Write the statement as if you were talking to a 3rd grader - this will help me get the "essence" of what each project does.